docs/feature/dualstack/Cilium-dualstack.md

```diff
@@ -0,0 +1,67 @@
+# Cilium Azure-ipam Dual Stack Overlay Proposal 
+
+## Purpose 
+
+This document proposes changes to the azure-ipam contract so that we can work with dual stack clusters between Cilium and Azure CNS. 
```

Comment:
```suggestion
This document proposes changes to the azure-cns contract so that we can work with dual stack clusters. 
```
the proposed changes are to the CNS API and only involve azure-ipam as a side-effect as a consumer of that API